sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/BaseAuthorizationTokenProvider.java

```diff
@@ -49,6 +49,7 @@ private static String getResourceSegment(ResourceType resourceType) {
         case Document:
             return Paths.DOCUMENTS_PATH_SEGMENT;
         case DocumentCollection:
+        case PartitionKey:
```

Comment:
is this intentional to return colls path for PartitionKey ?

Reply:
Yes, the .net PR returns the colls path for PartitionKey